Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/Version.java

```diff
@@ -12,7 +12,7 @@
     public Version(String version) {
         if(version == null)
             throw new IllegalArgumentException("Version can not be null");
-        if(!version.matches("[0-9]+(\\.[0-9]+)*"))
+        if(!version.matches("[0-9A-Za-z]+(\\.[0-9A-Za-z]+)*"))
```

Comment:
This makes sense on the validation, but the concern is the impact on the comparison behavior. If I'm playing interpreter correctly, this will result in the following versions being "equal": `1.1.0A` and `1.1.0B`. We need to surface this in a way that items that use Versions can detect whether they are comparable at run-time. And then make sure we use that when we do version comparison. So an `isComparable` that would return true if the version consists only of numbers and periods, and false otherwise. Then use that flag in the version loader to determine whether version comparison can be performed.